R/ai_movement.R

```diff
@@ -0,0 +1,111 @@
+
+#' AI movement
+#'
+#' @param x_pos x position of the player in the arena.
+#' @param y_pos y position of the player in the arena.
+#' @param arena_filename Path to the text file containing the arena.
+#' @details See http://shorturl.at/wyCNT
+#' @return
+#' @export
+ai_movement <- function(x_pos, y_pos, arena_filename) {
```

Comment:
Ouch, this is indeed a complex function! For now, it is good enough. We'll create an Issue to bring the cyclomatic complexity down :+1: